easybuild/easyconfigs/p/PyCharm/PyCharm-2017.2.3.eb

```diff
@@ -0,0 +1,19 @@
+name = 'PyCharm'
+version = '2017.2.3'
+easyblock = 'Tarball'
+
+homepage = 'https://www.jetbrains.com/pycharm/'
+description = """PyCharm Community Edition: Python IDE for Professional Developers"""
+
+toolchain = {'name': 'dummy', 'version': 'dummy'}
```

Comment:
I believe dummy:dummy toolchain was deprecated in favor of toolchain "SYSTEM". One can see it in action i.e. in PR #9095.

Reply:
thanks @jose-d  let me fix this

Reply:
made the fix see https://github.com/easybuilders/easybuild-easyconfigs/pull/9100/commits/a9e74faf1a6c25ed50c0a80c7d137d9db652816d